ui/src/api/api.tsx

```diff
@@ -46,6 +46,14 @@ export const fetchFeature = async (project: string, qualifiedName: string) => {
 };
 
 export const fetchProjectLineages = async (project: string) => {
+  return Axios
+    .get<IFeatureLineage>(`${ API_ENDPOINT }/v1/purview/${ purview }/projects/${ project }?code=${ token }`, {})
```

Comment:
it says SQL in API endpoint. But here it's still related purview. Can you fix it?

Reply:
This is intended right now. SQL based registration api is compatible with Purview based registration api. The purview account name will be passed but not used in SQL implementation. 

In a further version. Purview based registration api will remove purview accounts as parameter and move it to server side config. Then all purview related config will be removed in ui code as well.

Reply:
let's put a TODO here.